khelo/lib/ui/flow/profile/profile_screen.dart

```diff
@@ -189,15 +196,27 @@ class _ProfileScreenState extends ConsumerState<ProfileScreen>
             context,
             icon: Assets.images.icPrivacyPolicy,
             title: context.l10n.profile_setting_privacy_policy_title,
-            onTap: () => notifier
-                .openUrl("https://canopas.github.io/khelo/privacy-policy"),
+            onTap: () => notifier.openUrl(privacyPolicyUrl),
           ),
           _settingItem(
             context,
             icon: Assets.images.icTermsConditions,
             title: context.l10n.profile_setting_terms_and_condition_title,
-            onTap: () => notifier
-                .openUrl("https://canopas.github.io/khelo/terms-and-condition"),
+            onTap: () => notifier.openUrl(termsAndConditionsUrl),
+          ),
+          _settingItem(
+            context,
+            icon: Assets.images.icShare,
+            title: context.l10n.profile_setting_share_app_title,
+            onTap: () => notifier.onShareApp(context.l10n
```

Comment:
Share app not working and throw error in console like `Unhandled Exception: MissingPluginException(No implementation found for method share on channel dev.fluttercommunity.plus/share)`.

Also share app and rate us setting item icon not visible.
![Screenshot from 2024-11-20 11-55-36](https://github.com/user-attachments/assets/2ce89f41-e546-4ba9-a779-4165836e9703)